lib/friendly_id/base.rb

```diff
@@ -196,6 +196,7 @@ module Base
     def friendly_id(base = nil, options = {}, &block)
       yield friendly_id_config if block_given?
       friendly_id_config.dependent = options.delete :dependent
+      # friendly_id_config.routes = (options.delete(:routes) || :friendly)
```

Comment:
I'm a little confused about this line of code and why it's commented out? :smile: